docs/usage.md

```diff
@@ -69,119 +115,360 @@ cat ./compilation.json
   and in most cases, _System Integrity Protection_ needs to be turned off.
   See the [README](/README.md#mac-os-x) for details.
 
-## Step 2: Analyze your code <a name="step-2"></a>
-Once the build is logged successfully (and the `compilation.json`) was created,
-you can analyze your project.
+## Step 2: Analyze your code<a name="step-2"></a>
+Once the build is logged successfully and the `compile_commands.json` was
+created, you can analyze your project.
 
-1. Run the analysis:
+### Run the analysis<a name="run-the-analysis"></a>
 ```sh
-CodeChecker analyze compilation.json -o ./reports
+CodeChecker analyze ./compile_commands.json --enable-all --output ./reports
```

Comment:
Don't use the `--enable-all` option, because it is not recommend.
```suggestion
CodeChecker analyze ./compile_commands.json --output ./reports
```

Reply:
Changed to sensitive group.